front/src/views/Home.vue

```diff
@@ -74,36 +66,36 @@ export default {
             team: null,
             theme: null,
             gif: null,
-            giphyService: new GiphyService()
+            giphyService: new GiphyService(),
+            teamsService: null,
+            assignedTeam2: null
```

Comment:
we will have to find a better naming on that

Reply:
assignedTeamLocal ?

Reply:
I did another proposition in comments :)